jobs/involuntary-dissolutions/involuntary_dissolutions.py

```diff
@@ -73,7 +74,7 @@ def shell_context():
 
 
 def initiate_dissolution_process(app: Flask):  # pylint: disable=redefined-outer-name
-    """Initiate dissolution process for new businesses that meet dissolution criteria"""
+    """Initiate dissolution process for new businesses that meet dissolution criteria."""
     try:
         # check if batch has already run today
         batch_today = Batch.find_by(batch_type=Batch.BatchType.INVOLUNTARY_DISSOLUTION, start_date=datetime.today())
```

Comment:
You can remove the 'start_date' parameter here to fix the linting error.

I had originally planned to update the 'find_by' function in the 'batch' model but I think I will just use a query here like you do for 'batch_processing' on line 116. I'll update the logic part in my PR.